### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sbhhdp**
Specify the timezone in the readme

### CODE REVIEW

**Review by leios — APPROVED** [review]
Yup, this is worth it

**Comment by leios** [comment]
I was about to merge it, but do you mind if we say "Eastern" or "ET" instead of "EDT"? This course goes through November when we change to "EST"

**Comment by sbhhdp** [comment]
> I was about to merge it, but do you mind if we say "Eastern" or "ET" instead of "EDT"? This course goes through November when we change to "EST"

Go ahead please.

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you do so, please add punctuation, and please change the colour of the part y
 [Alan Edelman](http://math.mit.edu/~edelman), [David P. Sanders](http://sistemas.fciencias.unam.mx/~dsanders/), [Grant Sanderson](https://www.3blue1brown.com/about), & [James Schloss](https://eapsweb.mit.edu/people/jars)
 
 ## Logistics
-TR 2:30&ndash;3:30pm, online
+TR 2:30&ndash;3:30pm EDT, online
```
Comment (leios) [inline README.md:31]:
```suggestion
TR 2:30&ndash;3:30pm Eastern (ET), online
```

**Pull request merged**